AccountProviderSample/AccountProviderSample.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
So what is the logic with sln in this repository? I saw some high level sln and though that one is solution for all the projects. Are we changing that so that each sample has own solution? Or is it a mixture of both some use the high level sln and some samples have their own solution?